linera-views/src/common.rs

```diff
@@ -459,43 +459,6 @@ where
     }
 }
 
-/// Sometimes we need a serialization that is different from the usual one and
-/// for example preserves order.
-/// The {to/from}_custom_bytes has to be coherent with the Borrow trait.
-pub trait CustomSerialize {
-    /// Serializes the value
-    fn to_custom_bytes<C: Context>(&self) -> Result<Vec<u8>, ViewError>
-    where
```

Comment:
Not sure why we went for a `where` clause here because it's kinda wrong. (The trait becomes empty if the constraint is not satisfied.)

Reply:
This part is now obsolete due to other works being merged.